xilem_web/src/after_update.rs

```diff
@@ -0,0 +1,204 @@
+// Copyright 2023 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use xilem_core::{MessageResult, Mut, View, ViewId, ViewMarker};
+
+use crate::{DomNode, DomView, DynMessage, ViewCtx};
+
+pub struct AfterBuild<E, F> {
```

Comment:
These likely also need the generic vars `State` and `Action` as attribute `phantom: PhantomData<fn() -> (State, Action)>`

We had some inference issues with these two variables in the past, when e.g. using `Adapt` around such a type, as that needed explicit typing then...

(same for all the other views)